### ISSUE TITLE
discuss

### ISSUE THREAD

**Issue by jbenet** [the-grid/jsjob-ethereum#2]
i'd be interested in discussing this -- in one of our community hangouts, or a private one, in the next few weeks :)

**Comment by jonnor** [comment]
Hey @jbenet. I am up for that and prefer an open discussion. Do you have a date for next community hangout? Could not find on the IPFS calendar

**Comment by bergie** [comment]
@jbenet yup, open is fine for me as well. Though, I'm coming to San Francisco on Apr 10th so that'd provide a possibility for also meeting in person if you're around.
